formSchemas.js

```diff
@@ -18,9 +18,27 @@ const loginSchema = {
       options: [API.getMatches()],
       errorMessage: 'errors.login.code',
     },
+  }
+}
+const sinSchema = {
+  sin: {
+    customSanitizer: {
+      options: value => {
+        //We want to remove any spaces, dash or underscores
+        return value ? value.replace(/[ \-_]*/g, '') : value
+      },
+    },
+    isLength: {
+      errorMessage: 'errors.login.length',
```

Comment:
Should this be SIN specific, or is this just a default error message? (either is fine right now, in my opinion)

Reply:
Just noticed the error message for this is only for access codes: `Access code must be 8 characters`